rewrite-groovy/src/main/java/org/openrewrite/groovy/GroovyParser.java

```diff
@@ -62,7 +62,7 @@ public class GroovyParser implements Parser {
 
     @Override
     public Stream<SourceFile> parse(@Language("groovy") String... sources) {
```

Comment:
Maybe we should drop that `@Language` annotation?
```suggestion
    public Stream<SourceFile> parse(String... sources) {
```

Reply:
Yes. I was just thinking that now it could both be Groovy or Kotlin, but I guess we can also leave it as is.

Reply:
Ahh, fair enough. The suggestion was on the Groovy language parser, so I was a little confused. The `GradleParser` doesn't override the `parse(String... sources)` method, so it doesn't specify the language at all. `GradleParser` acts as the entry point for all Gradle scripts, then delegates to either a `GroovyParser` or `KotlinParser` based upon the Gradle semantics.